src/app/content/utils/urlUtils.ts

```diff
@@ -91,6 +93,9 @@ export const toRelativeUrl = (from: string, to: string) => {
   // remove the last piece of the "to" so that it is always output
   const commonParts = getCommonParts(parsedFrom, parsedTo.slice(0, -1));
 
-  return '../'.repeat(parsedFrom.length - commonParts.length - 1)
+  const lastFromFragment = last(parsedFrom);
+  const hasTrailingSlash = lastFromFragment && last(parsedTo) !== lastFromFragment && /\/$/.test(from);
+
+  return '../'.repeat(parsedFrom.length - commonParts.length - 1 + (hasTrailingSlash ?  1 : 0))
```

Comment:
This would be a little bit easier to read if it was something like this:
```
const hasTrailingSlash = /\/$/.test(from);
const valueToSubtract = hasTrailingSlash && (last(parsedTo) !== last(parsedFrom)
  ? 0
  : 1
return '../'.repeat(parsedFrom.length - commonParts.length - valueToSubtract)
...
```
`